notification/hub.js

```diff
@@ -0,0 +1,25 @@
+var authorize = require('socket-server/cookie-authorize');
+var db = require('firebase-db');
+var ref = db.ref('notification');
+
+var socketMap = {};
+
+var subscribe = function (socket) {
+  authorize(socket).then(function (username) {
```

Comment:
You could break this out using the socketio-jwt middleware as discussed on gitter